pom.xml

```diff
@@ -426,6 +426,7 @@
         <version.org.jboss.mod_cluster>1.4.3.Final</version.org.jboss.mod_cluster>
         <version.org.jboss.narayana>5.12.0.Final</version.org.jboss.narayana>
         <version.org.jboss.openjdk-orb>8.1.5.Final</version.org.jboss.openjdk-orb>
+        <version.org.picketbox>5.0.3.Final-redhat-00007</version.org.picketbox>
```

Comment:
Hi @darranl , I was taking a look in this change and seems that wildfly pom already contained the org.picketbox property.

So now it is duplicated, see both https://github.com/wildfly/wildfly/blob/main/pom.xml#L432 and https://github.com/wildfly/wildfly/blob/main/pom.xml#L478

Reply:
We will have a further look in our next round of removals, ideally all PicketBox references get to be removed but we will see.